pkg/stream/edgedlogconnection.go

```diff
@@ -36,10 +36,22 @@ type EdgedLogsConnection struct {
 	Header   http.Header   `json:"header"`
 }
 
+func (l *EdgedLogsConnection) CreateConnectMessage() (*Message, error) {
+	data, err := json.Marshal(l)
+	if err != nil {
+		return nil, err
```

Comment:
Codecov is asking me to cover case where `json.Marshal` gives an error. I think tests are for testing logic of the functions and not standard libraries like `json.Marshal` (how can I make `json.Marshal` give error?). what do you guys have to say about it ? (also I haven't created this func I have simply moved its place so that all funcs are consistently placed in all files)

Reply:
I think we needn't to cover this case.